yammbs/analysis.py

```diff
@@ -124,13 +124,12 @@ def get_rmsd(
     )
 
 
-def get_internal_coordinate_rmsds(
+def get_internal_coordinates(
```

Comment:
Need a docstring. It would be useful as a user to know where I'm getting these inputs, e.g., molecule is from the openff_toolkit. Also that `reference` and `target` are both conformers in `molecules` and that all conformers in `molecule` will be discarded, and replaced with either `reference` or `target`. 

This seems like an interesting choice in inputs and that one could likely get around having `molecule` as an input but there's precedent, so I expect it's difficult to change at this point.